README.md

```diff
@@ -1,4 +1,4 @@
-MyExpenses
+MyExpenses  [![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=267118)](https://www.bountysource.com/trackers/267118-myexpenses?utm_source=267118&utm_medium=shield&utm_campaign=TRACKER_BADGE)
```

Comment:
Thanks for the pull request. Why do you think the bountysource link should be repeated in the title? I was rather thinking about removing it all together, since there has not been any activity with respect to this project on Bounty Source for a long time.